documentation/source/index.rst

```diff
@@ -1,37 +1,158 @@
-**********************************
+##################################
 Welcome to Cocotb's documentation!
-**********************************
+##################################
 
-Contents:
+..
+   This documentation tries to follow https://www.divio.com/blog/documentation/ (Daniele Procida)
+   Other media about the same topic:
+   - https://ep2018.europython.eu/media/conference/slides/get-your-documentation-right.pdf
+   - https://www.youtube.com/watch?v=t4vKPhjcMZg
+   - A good example: http://docs.django-cms.org/en/latest/contributing/documentation.html#contributing-documentation
+
+   See also https://github.com/cocotb/cocotb/wiki/Howto:-Writing-Documentation
 
 .. toctree::
    :maxdepth: 2
 
    introduction
+
+.. todo::
+   Move "Contributors" section to "Development & Community"
+
+#########
+Tutorials
```

Comment:
You might want to make this a `:caption:` of the TOC so it appears in the sidebar.

Reply:
Ah yes, so much nicer!